cishouseholds/validate.py

```diff
@@ -1,10 +1,41 @@
 import csv
+from datetime import datetime
+
+from cerberus import TypeDefinition
+from cerberus import Validator
+from pyspark.sql import Row
+
+from cishouseholds.pyspark_utils import ListAccumulator
 
 
 class InvalidFileError(Exception):
     pass
 
 
+class PysparkValidator(Validator):
+    types_mapping = Validator.types_mapping.copy()
+    types_mapping["timestamp"] = TypeDefinition("timestamp", (datetime,), ())
```

Comment:
Bit of a pain, but based on our experience on Wednesday, is it worth incorporating the timestamp type into the later tests for filtering validation?